src/app/tests/suites/certification/Test_TC_DISHM_2_1.yaml

```diff
@@ -80,106 +80,273 @@ tests:
           [1687687925.445365][18054:18056] CHIP:TOO: Value: 2
           [1687687925.445385][18054:18056] CHIP:TOO: }
           [1687687925.445405][18054:18056] CHIP:TOO: }
-      disabled: true
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_USER_PROMPT
+      arguments:
+          values:
+              - name: "message"
+                value: "Please enter 'y' for success"
+              - name: "expectedValue"
+                value: "y"
 
-    - label: "Step 2: TH reads from the DUT the CurrentMode attribute."
-      PICS: DISHM.S.A0001
+    - label: "TH reads the CurrentMode attribute from the DUT"
       verification: |
-          ./chip-tool dishwashermodeselect read current-mode 1 1
+          ./chip-tool dishwashermode read current-mode 1 1
 
           Verify on TH(chip-tool) logs, CurrentMode attribute value is an integer value
           - Save the value as old_current_mode_dut
-          - Select a value from supported_modes_dut, different from old_current_mode_dut, and which corresponds to a mode the device can switch to, given its current state and save this value as new_mode_th
           - Select a value which is NOT in supported_modes_dut and save it as invalid_mode_th,
```

Comment:
I guess this "select a value" bit is the part that keeps this test from being automated?

Reply:
Yes, that was the issue I ran into when writing and attempting to run the code regeneration